backend/fms_core/template_importer/row_handlers/library_conversion/library_conversion.py

```diff
@@ -0,0 +1,67 @@
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+from fms_core.services.sample import get_sample_from_container
+from fms_core.services.container import get_container, get_or_create_container
+from fms_core.services.library import convert_library
+
+
+
+class LibraryRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+    def process_row_inner(self, library_batch_info, library_source, volume_used, comment,
+                          container, concentration, library_size, volume):
+
+        if not library_batch_info:
+            self.errors['library_conversion'] = 'No batch is associated with this library.'
+
+        # Calling the service creator for Samples in LibraryPreparation
+        sample_source_obj, self.errors['container'], self.warnings['container'] = \
+            get_sample_from_container(barcode=library_source['barcode'], coordinates=library_source['coordinates'])
+
+        if sample_source_obj:
+            # Populate the libraries with the batch and  individual information
+            container_coordinates = container['coordinates']
+
+            container_parent_obj = None
+            if container['parent_barcode']:
+                container_parent_obj, self.errors['parent_container'], self.warnings['parent_container'] = \
+                    get_container(barcode=container['parent_barcode'])
+
+            container_obj, created, self.errors['library_container'], self.warnings['library_container'] = get_or_create_container(
+                name=container['name'],
+                barcode=container['barcode'],
+                kind=container['kind'],
+                container_parent=container_parent_obj if container_parent_obj else None,
+                coordinates=container['parent_coordinates'] if container_parent_obj else None,
+                creation_comment=comment)
```

Comment:
I don't think this comment is meant for the container creation ... You would be better to add an automatic message. Something like : Container created automatically while doing library conversion + timestamp.